### PR TITLE
update version for m134

### PR DESCRIPTION
**Pull request by VinayGuthal**
NO_RELEASE_CHANGE

### CODE REVIEW

File: firebase-config/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Unreleased
+# 21.4.1
```
Comment (rlazo) [inline firebase-config/CHANGELOG.md:2]: Add an empty line above

**Comment by rlazo** [comment]
Add 

```
NO_RELEASE_CHANGE
```

to your description

File: firebase-firestore/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,16 @@
 # Unreleased
+* [fixed] Implement equals method on Filter class. [#5210](//github.com/firebase/firebase-android-sdk/issues/5210)
+
+# 24.7.0
 * [feature] Expose MultiDb support in API. [#4015](//github.com/firebase/firebase-android-sdk/issues/4015)
 * [fixed] Fixed a thread interference issue that may lead to a ConcurrentModificationException.
-    (GitHub [#5091](//github.com/firebase/firebase-android-sdk/issues/5091){: .external})
-* [fixed] Implement equals method on Filter class. [#5210](//github.com/firebase/firebase-android-sdk/issues/5210)
+  (GitHub [#5091](//github.com/firebase/firebase-android-sdk/issues/5091){: .external})
```
Comment (emilypgoogle) [inline firebase-firestore/CHANGELOG.md:7]: Is this change of indentation intentional?

File: firebase-messaging/CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,21 @@
 # Unreleased
-* [deprecated] Deprecated FCM upstream messaging. See the
-  [FAQ](https://firebase.google.com/support/faq#fcm-23-deprecation) for more
-  details.
+
```
Comment (emilypgoogle) [inline firebase-messaging/CHANGELOG.md:2]: Isn't there not supposed to be a blank line here?

File: firebase-firestore/CHANGELOG.md (hunk 2 above)
Comment (VinayGuthal) [reply firebase-firestore/CHANGELOG.md:7]: it was just a copy paste but in hind sight i think the current one is the correct identation

File: firebase-messaging/CHANGELOG.md (hunk 3 above)
Comment (VinayGuthal) [reply firebase-messaging/CHANGELOG.md]: Removed the blank line

**Review by emilypgoogle — APPROVED** [review]

**Review by rlazo — APPROVED** [review]

**Pull request merged**